### DISCUSSION TITLE
Query and use Saved Searches in my iTwin viewer

### DISCUSSION THREAD

**Discussion by JohnMajerle** [iTwin/itwinjs-core#5550] [Q&A]
Using the following dialog below I'd like to create a variety of Saved Searches that I would later on use to query for elements from my iTwin viewer application.  Is there an API to retrieve these?

![image](https://github.com/iTwin/itwinjs-core/assets/30668739/6fc20d93-28b5-453c-aa75-f14fc8c3dec3)

**Comment by pratikshan85** [comment]
@JohnMajerle  We are looking into your query.

**Comment by aruniverse** [comment]
Hi John, 

At the moment, we don't have a public API for this, but we will look into it.

**Comment by JohnMajerle** [comment]
@aruniverse - An even better approach for us might be for you to create a sandbox sample based on the Advanced Search form.  That way I could both bring it up from my web application and I could add some enhancements that would better fit our workflow.
